filebrowser.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: filebrowser
+  version: "2.31.2"
+  description: "Web File Browser"
+  epoch: 0
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - npm
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/filebrowser/filebrowser.git
+      tag: v${{package.version}}
+      expected-commit: 129a4fd39d550bd2a449575cf93f061593e39102
+
+  - uses: go/bump
+    with:
+      deps: github.com/golang-jwt/jwt/v4@v4.5.1
+
+  - name: install-npm-dependencies
+    working-directory: frontend
+    pipeline:
+      - runs: |
+          npm install --frozen-lockfile
+          NODE_ENV=production npm run build
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: .
+      ldflags: |
+        -X github.com/filebrowser/filebrowser/v2/version.Version=${{package.version}}
+        -X github.com/filebrowser/filebrowser/v2/version.CommitSHA=$(git rev-parse HEAD)
+      output: ${{package.name}}
+
+  - name: copy files
+    pipeline:
+      - runs: |
+          cp docker_config.json ${{targets.contextdir}}/.filebrowser.json
+          cp healthcheck.sh ${{targets.contextdir}}/
+
+test:
+  environment:
+  pipeline:
+    - runs: |
+        filebrowser --help
+        filebrowser version | grep ${{package.version}}
```

Comment:
Is it possible to expand the tests a bit?

Reply:
Checking the health status as additional test.